extensions/reviewed/Gamepads.json

```diff
@@ -1465,6 +1629,12 @@
             "            16: \"PS_BUTTON\",\r",
             "            17: \"CLICK_TOUCHPAD\"\r",
             "        }\r",
+            "    },\r",
+            "    rumble: {\r",
+            "        duration: 0,\r",
+            "        elapsedTime: 0,\r",
+            "        weakMagnitude: 0,\r",
+            "        strongMagnitude: 0\r",
```

Comment:
Since the rumble was moved into `gdjs._extensionController.players`, this is now redundant.
Other than that, the code seems good enough to me. For now, it's probably best to wait for ZachjuKamashi to give feedback on whether or not the rumble works as expected with multiple controllers.